pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -149,6 +152,37 @@ func (rs *RevisionSpec) Validate(ctx context.Context) *apis.FieldError {
 	return errs
 }
 
+func validateAnnotations(annotations map[string]string) *apis.FieldError {
+	return validatePercentageAnnotationKey(annotations, serving.QueueSideCarResourcePercentageAnnotation)
+}
+
+func validatePercentageAnnotationKey(annotations map[string]string, resourcePercentageAnnotationKey string) *apis.FieldError {
+	if len(annotations) == 0 {
+		return nil
+	}
+
+	v, ok := annotations[resourcePercentageAnnotationKey]
+	if !ok {
+		return nil
+	}
+	value, err := strconv.ParseFloat(v, 32)
```

Comment:
probably not a float anymore?

Reply:
As discussed with @mattmoor  we still want to support 10.5% so keeping as float